src/components/regionalization/RegionalizationModal/regionalization-modal.scss

```diff
@@ -0,0 +1,100 @@
+@import "src/styles/scaffold";
+
+[data-modal-overlay] {
+  position: fixed;
+  top: 0;
+  right: 0;
+  bottom: 0;
+  left: 0;
+  z-index: 3;
+  background-color: rgb(0 0 0 / 25%);
```

Comment:
It's such a slight difference 🤔 Maybe we could double-check with the design team this background opacity.